src/sentry/issues/grouptype.py

```diff
@@ -483,12 +483,21 @@ class MonitorCheckInMissed(GroupType):
 
 @dataclass(frozen=True)
 class ReplayDeadClickType(GroupType):
```

Comment:
We can't delete this right?

Reply:
not sure about consequences of deleting here.

Reply:
Yeah it's fine to leave it.  Maybe mark it as a todo and ask issues team if you're feeling froggy lol.